waiter/src/waiter/client_tools.clj

```diff
@@ -519,13 +519,14 @@
       (str/replace (str test-prefix service-name username (rand-int 3000000)) #"-" ""))))
 
 (defn delete-token-and-assert
-  [waiter-url token]
+  [waiter-url token & {:keys [query-params] :or {query-params {}}}]
```

Comment:
What about erasing tokens created in tests that you haven't updated?  Should `erase` be on by default for unit tests?